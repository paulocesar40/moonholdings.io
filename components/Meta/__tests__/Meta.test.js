import React from 'react'
import Head from 'next/head'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Meta from '../Meta'

describe('<Astronaut /> component', () => {
  describe('rendering', () => {
    const wrapper = shallow(<Meta />);
    it('should render a component matching the snapshot', () => {
      const tree = toJson(wrapper);
      expect(tree).toMatchSnapshot();
      expect(wrapper).toHaveLength(1);
      expect(wrapper.contains(<Head/>));
    });
  });
});
