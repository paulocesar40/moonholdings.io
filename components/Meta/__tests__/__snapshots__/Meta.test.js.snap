// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Astronaut /> component rendering should render a component matching the snapshot 1`] = `
<SideEffect(Head)>
  <title>
    MOON.HOLDINGS
  </title>
  <meta
    content="IE=edge"
    httpEquiv="X-UA-Compatible"
  />
  <meta
    content="initial-scale=1.0, width=device-width"
    name="viewport"
  />
  <meta
    content="Your Cryptocurrency Portfolio"
    name="description"
  />
  <meta
    content="cryptocurrency, crypto, portfolio, bitcoin, ethereum, holdings"
    name="keywords"
  />
  <meta
    content="index, follow"
    name="robots"
  />
  <meta
    content="https://twitter.com/leongaban"
    name="owner"
  />
  <meta
    content="https://moon.holdings"
    name="url"
  />
  <meta
    content="https://moon.holdings"
    name="identifier-URL"
  />
  <meta
    content="finance"
    name="category"
  />
  <meta
    content="General"
    name="rating"
  />
  <meta
    content="7 days"
    name="revisit-after"
  />
  <meta
    content="MOON.HOLDINGS"
    name="og:title"
  />
  <meta
    content="app"
    name="og:type"
  />
  <meta
    content="https://moon.holdings"
    name="og:url"
  />
  <meta
    content="https://moon.holdings/static/logo.png"
    name="og:image"
  />
  <meta
    content="MOON.HOLDINGS"
    name="og:site_name"
  />
  <meta
    content="Your Cryptocurrency Portfolio"
    name="og:description"
  />
  <meta
    content="MOON.HOLDINGS"
    name="twitter:title"
  />
  <meta
    content="Your Cryptocurrency Portfolio"
    name="twitter:description"
  />
  <meta
    content="https://moon.holdings/static/twittercard.png"
    name="twitter:image"
  />
  <meta
    content="https://moon.holdings/static/twittercard.png"
    name="twitter:card"
  />
  <link
    href="static/favicon.ico"
    rel="icon"
    type="image/x-icon"
  />
  <link
    href="static/apple-icon-180x180.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />
  <link
    href="static/android-icon-192x192.png"
    rel="icon"
    sizes="192x192"
    type="image/png"
  />
  <link
    href="https://fonts.googleapis.com/css?family=Abel|Overpass:200,700"
    rel="stylesheet"
  />
</SideEffect(Head)>
`;
